package.json

```diff
@@ -36,8 +38,17 @@
   },
   "devDependencies": {
     "@stencil/core": "^1.3.0",
+    "@manifoldco/eslint-plugin-stencil": "^0.4.1",
     "@types/jest": "24.0.17",
     "@types/puppeteer": "1.19.1",
+    "@typescript-eslint/eslint-plugin": "^2.0.0",
+    "@typescript-eslint/parser": "^2.0.0",
+    "eslint": "^6.2.1",
+    "eslint-config-airbnb-base": "^14.0.0",
```

Comment:
I never noticed until seeing this that we were using an airbnb config in UI as well. Is that the standard for a Stencil project or was there a different reason we decided to use it?

Reply:
That’s the standard. It’s just the base, so it’s JS style not JSX